nexpose/nexpose_site.py

```diff
@@ -176,6 +180,38 @@ def AsXML(self, exclude_id):
         xml_scanconfig.append(xml_scheduling)
         xml_data.append(xml_scanconfig)
 
+        xml_tags = create_element('Tags')
+        for tag in self.tags:
+            for tag_data in tag:
+                tag_data_name = get_attribute(tag_data, 'name')
+                tag_data_value = get_attribute(tag_data, 'value')
+
+                # CREATION_DATE must be in epoch time.
+                if tag_data_name == 'CREATION_DATE':
+                    creation_date_value = get_attribute(tag_data, 'value')
+                    creation_date_as_datetime = datetime.datetime.strptime(creation_date_value, '%a %b %d %H:%M:%S %Z %Y')
```

Comment:
The date format string could be extracted to a constant, but I don't think it's a big deal.